### ISSUE TITLE
Validate on LFW with 20161030 accuracy is 67%

### ISSUE THREAD

**Issue by apollo-time** [davidsandberg/facenet#72]
I  test validate on LFW with pre-trained model.
OS : Ubuntu 14 64bit. CPU mode.

~/work/facenet/facenet/src$ python validate_on_lfw.py ~/work/facenet/facenet/data/lfw_mtcnnpy_182 ~/work/facenet/facenet/model/20161030-023650 --lfw_pairs=~/work/facenet/facenet/data/pairs.txt 
Model directory: /home/apollo/work/facenet/facenet/model/20161030-023650
Metagraph file: model-20161030-023650.meta
Checkpoint file: model-20161030-023650.ckpt-80000
Runnning forward pass on LFW images
Accuracy: 0.678+-0.016
Validation rate: 0.05905+-0.01552 @ FAR=0.00131

What did I wrong?

**Comment by davidsandberg** [comment]
It usually looks like this when running Validate on LFW on unaligned images.
Did you align the LFW images as described in [Validate on LFW](https://github.com/davidsandberg/facenet/wiki/Validate-on-LFW)?

**Comment by apollo-time** [comment]
Yes, I did align it.
But I got previous result with phase_train_placeholder set False.
And I try test with phase_train_placeholder set True, I got good result.

Runnning forward pass on LFW images
Accuracy: 0.979+-0.008
Validation rate: 0.91245+-0.01793 @ FAR=0.00136

What's the matter?
When test graph without train, is setting phase_train_placeholder  with True Ok?
Moreover If I want to use this graph in really face feature extraction, then is phase_train_placeholder True or False?
